rllib/examples/env/mock_env.py

```diff
@@ -34,6 +35,10 @@ class MockEnv2(gym.Env):
     configurable. Actions are ignored.
     """
 
+    metadata = {
```

Comment:
why do we need to do this?

Reply:
I dunno. :D 
Gym requires this. Otherwise, the gym Video wrapper won't call the env's `render` method :(

Reply:
oh good catch, I didn't know that. Should we add this as an overridable in the base env class?